FindOpenGLES.cmake

```diff
@@ -0,0 +1,60 @@
+#.rst:
+# FindOpenGLES
+# ------------
+# Finds the OpenGLES2 library
```

Comment:
finds OpenGLES2 and OpenGLES3

Reply:
A remnant where that came from Kodi: https://github.com/xbmc/xbmc/blob/master/cmake/modules/FindOpenGLES.cmake#L4